src/app/app.html

```diff
@@ -9,6 +9,10 @@
     <ion-content>
       <ion-list role="navigation" id="navigation-menu" aria-label="Peeveetrack Menu">
         <button role="link" menuClose ion-item *ngFor="let p of pages" (click)="openPage(p)">
+        <ion-icon name="ios-heart-outline" *ngIf="p.title == 'Favorites'"></ion-icon>
```

Comment:
I wonder if I could put this in the typescript. Not advanced enough to do something like that. But I can try.